### PR TITLE
Grid viewpager 2 running as expected

### PR DESCRIPTION
**Pull request by samirtf**

### CODE REVIEW

File: GridViewPager2/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/btn1X1"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="1X1"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/activity_main.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
+        android:id="@+id/btn2X2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="2X2"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/activity_main.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        app:layout_constraintStart_toEndOf="@+id/btn1X1" />
+
+    <Button
+        android:id="@+id/btn3X3"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="3X3"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/activity_main.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    <Button
+        android:id="@+id/btn4X4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="4X4"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/activity_main.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category.xml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        android:id="@+id/item01"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="TextView"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_1x1.xml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        android:id="@+id/item01"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="TextView"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_1x1.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    android:background="@color/colorAccent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/item01"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorA"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="A"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item02"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorB"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="B"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+        app:layout_constraintStart_toEndOf="@+id/item01"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item03"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorC"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="C"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item01" />
+
+    <TextView
+        android:id="@+id/item04"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorD"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="D"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_2x2.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+    android:background="@color/colorAccent"
+    android:orientation="vertical">
+
+
+    <TextView
+        android:id="@+id/item01"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="A"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item02"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="B"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item01"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item03"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="C"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item02"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item04"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="D"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item01" />
+
+    <TextView
+        android:id="@+id/item05"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="E"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item04"
+        app:layout_constraintTop_toBottomOf="@+id/item02" />
+
+    <TextView
+        android:id="@+id/item06"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="F"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item05"
+        app:layout_constraintTop_toBottomOf="@+id/item03" />
+
+    <TextView
+        android:id="@+id/item07"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="G"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item04" />
+
+    <TextView
+        android:id="@+id/item08"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="H"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml [hunk 19]
```diff
@@ -0,0 +1,145 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item07"
+        app:layout_constraintTop_toBottomOf="@+id/item05" />
+
+    <TextView
+        android:id="@+id/item09"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="I"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_3x3.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 20]
```diff
@@ -0,0 +1,257 @@
...
+    android:background="@color/colorAccent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/item01"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorA"
+        android:gravity="center_horizontal|center_vertical"
+        android:text="A"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 21]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintEnd_toStartOf="@+id/item02"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item02"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:text="B"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 22]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintEnd_toStartOf="@+id/item03"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item01"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item03"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:text="C"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 23]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item02"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item04"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorD"
+        android:text="D"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 24]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item03"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/item05"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorE"
+        android:text="E"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 25]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item01" />
+
+    <TextView
+        android:id="@+id/item06"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorF"
+        android:text="F"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 26]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item05"
+        app:layout_constraintTop_toBottomOf="@+id/item02" />
+
+    <TextView
+        android:id="@+id/item07"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorG"
+        android:text="G"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 27]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item06"
+        app:layout_constraintTop_toBottomOf="@+id/item03" />
+
+    <TextView
+        android:id="@+id/item08"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorH"
+        android:text="H"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 28]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item07"
+        app:layout_constraintTop_toBottomOf="@+id/item04" />
+
+    <TextView
+        android:id="@+id/item09"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorI"
+        android:text="I"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 29]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item05" />
+
+    <TextView
+        android:id="@+id/item10"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorJ"
+        android:text="J"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 30]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item09"
+        app:layout_constraintTop_toBottomOf="@+id/item06" />
+
+    <TextView
+        android:id="@+id/item11"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorK"
+        android:text="K"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 31]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item10"
+        app:layout_constraintTop_toBottomOf="@+id/item07" />
+
+    <TextView
+        android:id="@+id/item12"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorL"
+        android:text="L"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 32]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item11"
+        app:layout_constraintTop_toBottomOf="@+id/item08" />
+
+    <TextView
+        android:id="@+id/item13"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorM"
+        android:text="M"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 33]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/item09" />
+
+    <TextView
+        android:id="@+id/item14"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorN"
+        android:text="N"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 34]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item13"
+        app:layout_constraintTop_toBottomOf="@+id/item10" />
+
+    <TextView
+        android:id="@+id/item15"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorO"
+        android:text="O"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml [hunk 35]
```diff
@@ -0,0 +1,257 @@
...
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/item14"
+        app:layout_constraintTop_toBottomOf="@+id/item11" />
+
+    <TextView
+        android:id="@+id/item16"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/colorP"
+        android:text="P"
```
Comment (cesar-ferreira) [inline GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: extracting text to string file may be a good idea to reuse it

**Review by cesar-ferreira — COMMENTED** [review]
extracting text to string file may be a good idea to reuse it

File: GridViewPager2/app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/activity_main.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/activity_main.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml (hunk 31 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/item_category.xml (hunk 5 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/item_category.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml (hunk 32 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml (hunk 33 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml (hunk 34 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: Done.

File: GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml (hunk 35 above)
Comment (samirtf) [reply GridViewPager2/app/src/main/res/layout/item_category_container_4x4.xml]: Done.

**Review by cesar-ferreira — APPROVED** [review]
Reviewed by @cesar-ferreira

**Pull request merged**